Automattic/simplenote-macos
# Update about view to match other apps

The other apps had their about views updated last year. macOS still needs it! Here's what it looks like:

<img width="531" alt="screen shot 2018-02-07 at 3 29 40 pm" src="https://user-images.githubusercontent.com/789137/35948171-275e0fd2-0c20-11e8-833d-2b103c890f79.png">

Some notes on the code: Click events on `NSTextField` objects are actually kind of a pain. I had to do some tricks in the code to get the `NSTextFields` to be clickable (Using `NSClickGestureRecognizer`) and also have a hand pointer icon upon hover (using a subclass).

It also dynamically displays the copyright year and the version of the app 🚀 

I've added a story board and also [got Swifty](https://www.youtube.com/watch?v=I1188GO4p1E)!

**To Test**
* Select Simplenote-About.
* Observe the new screen and click on all the things and ensure they load the correct URL.
* Try opening multiple windows, it should only allow one.
* Bring the main Simplenote window in front of the about view. Selecting Simplenote->About again should bring it to the front.
